samples/bluetooth/nrf_dm/README.rst

```diff
@@ -78,6 +78,22 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Energy consumption and the number of measurements performed in a given time depend on:
+
+   * Advertiser settings.
+   * Scanner settings.
+   * Extended size of measurement window by ``extra_window_time_us`` in the :c:struct:`dm_request` structure.
+
+The shorter the interval, the higher the frequency of broadcasting advertising packages.
+This leads to a higher probability of synchronization and to a higher number of measurements taken at a given time.
+However, the higher number of packets transmitted also results in higher power consumption.
+
+The scan interval and scan window parameters determine how often and how long the scanning device will listen for packets.
+Like the short advertising interval, these values also have an impact on energy consumption.
+
+Extending the measurement window will increase the probability of measurement success for coarse synchronization.
+However, it affects the radio's turn-on time and thus, power consumption.
+
```

Comment:
Maybe we can also mention DM_MIN_TIME_BETWEEN_TIMESLOTS_US and DM_TIMESLOT_QUEUE_COUNT_SAME_PEER with which you can control how often the same peer is ranged.